evalme/classification.py

```diff
@@ -96,7 +98,7 @@ class ChoicesEvalItem(ClassificationEvalItem):
 
 
 class PairwiseEvalItem(ClassificationEvalItem):
-    SHAPE_KEY = 'pairwise'
+    SHAPE_KEY = 'selected'
```

Comment:
this seems pretty strange, are we pretty confident that this should definitely be the value here? And that there isn't something else depending on it being "pairwise"? (maybe worth leaving a comment about why it's "selected" instead of "pairwise"?)

Reply:
Whats strange is that it should have been 'selected' this entire time and it was not caught because we have a try/except that was silencing the error. However when comparing an empty annotation to a GT the error occurs, and we dont make such comparisons in normal LS it seems. But I will make sure this doesnt affect anything else

Reply:
This was added 5 years ago and perhaps the key that we used in annotations back then was 'pairwise' but it most definitely is not now. It is 'selected' 
This tells me that no one has noticed that metrics/scores are wrong for pairwise tag comparisons lol 

https://github.com/HumanSignal/label-studio-evalme/commit/f4408434af8e736fa1585234a53ad70a97ba39f1

Reply:
![image](https://github.com/user-attachments/assets/b680f3f2-1f66-498e-b809-0b1ff3a0dd24)